app/server/app/routes/data.js

```diff
@@ -14,15 +161,18 @@ module.exports = function (app) {
 
   // --- get assessments from database
   router.get("/assessments", function (req, res, next) {
-    try {
-      return Assessment.schema(req.activeSchema)
-        .findAndCountAll({})
-        .then((assessments) => res.status(200).send(assessments))
-        .catch((error) => res.status(500).send("Error! " + error));
-    } catch (error) {
-      log.error("Failed to get assessments...");
-      return res.status(500).send("Error !" + error);
-    }
+    executeQuery("assessments", req, res, next);
+  });
+  router.get("/assessments/count", function (req, res, next) {
+    executeQueryCountOnly("assessments", req, res, next);
+  });
+
+  // --- get profile_test from database
+  router.get("/profileTests", function (req, res, next) {
+    executeQuery("profile_test", req, res, next);
+  });
+  router.get("/profileTests/count", function (req, res, next) {
+    executeQueryCountOnly("profile_test", req, res, next);
```

Comment:
I realize this is the test table, but for REST resources it's conventional to use hyphens to separate words. If we care about SEO for these routes, this will also help since Google views hyphens as word separators. For the filenames and worksheets, it could be helpful to use the resource name instead of the table name.

Reply:
EPA decided during the last meeting that we should copy attains and do camel casing (i.e., `profileTests` or `usesStateSummary`) for the endpoint names and parameter names. 

https://www.epa.gov/sites/default/files/2020-10/documents/attains_how_to_access_web_services_2020-10-28.pdf

Reply:
Sounds good! I wasn't sure if we'd decided on that just for the parameters or for the endpoints, too.